lib/tasks/database.rake

```diff
@@ -0,0 +1,74 @@
+require "mysql2"
+require "yaml"
+require "nokogiri"
+require "reverse_markdown"
+
+namespace :database do
+  namespace :legacy do
+    # NOTE to run remotely on heroku:
+    # Source: https://devcenter.heroku.com/articles/rake
+    #
+    # $ heroku run rake database:legacy:import_news
+    #
+    desc "Import news records to database"
+    task :import_news => :environment do
+      STDOUT.puts "This will destroy your news_items table. Enter 'Y' to confirm: [y/N]"
+      input = STDIN.gets.chomp
+      raise "Aborting import. You entered #{input}" unless input.downcase == "y"
+
+      NewsItem.destroy_all
+      records = YAML.load_file "#{Rails.root}/db/legacy.yml"
+      records.each do |data|
+        news_item = NewsItem.new
+        published_at = data["post_date"]
+
+        news_item.state        = published_at > DateTime.new(2015, 1, 1).beginning_of_year ? :published : :archived
```

Comment:
The only reason why I am doing this, is because I wanted to automatically mark as published the 2015 posts, leaving us time to clean up old posts one by one.